### PR TITLE
Update MCR.cpp

### PR DESCRIPTION
**Pull request by yyxzz2**

### CODE REVIEW

**Review by yyxzz2 — COMMENTED** [review]
代码结束

**Review by zwdconstrucition — COMMENTED** [review]
#zwd_review

File: MCR.cpp [hunk 1]
```diff
@@ -56,3 +56,4 @@ cout << game[1][0] << " " << game[1][1] << " " << game[1][2] << endl;
 cout << game[2][0] << " " << game[2][1] << " " << game[2][2] << endl;
 return 0;
 }
+#12313
```
Comment (lowen2022) [inline MCR.cpp]: I see·

File: MCR.cpp [hunk 2]
```diff
@@ -56,4 +56,4 @@ cout << game[1][0] << " " << game[1][1] << " " << game[1][2] << endl;
 cout << game[2][0] << " " << game[2][1] << " " << game[2][2] << endl;
 return 0;
 }
-#12313
+#1231321
```
Comment (lowen2022) [inline MCR.cpp]: better one

**Review by lowen2022 — APPROVED** [review]

**Pull request merged**